Sok-O-Vando/src/main/java/org/academiadecodigo/asynctomatics/sokovando/TerminationSwitch.java

```diff
@@ -0,0 +1,8 @@
+package org.academiadecodigo.asynctomatics.sokovando;
+
+public class TerminationSwitch {
+
+    public void terminateGame () {
+        System.exit(0);
```

Comment:
nice step towards a better abstraction. In the future this can display some goodbye message.